src/minicli/builtins.go

```diff
@@ -280,6 +281,34 @@ func parseFilter(s string) (filter, error) {
 	return filter{}, errors.New("invalid filter, see help")
 }
 
+func findColumn(headers []string, column string) (int, error) {
+	foundI := -1
+	for i, header := range headers {
+		// if it's an exact match, don't check any further for collisions
+		if strings.ToLower(header) == column {
+			return i, nil
+		}
+
+		if !strings.HasPrefix(strings.ToLower(header), column) {
+			continue
+		}
+
+		if foundI >= 0 {
+			// collision
+			return 0, fmt.Errorf("ambiguous column `%s`", column)
+		}
+
+		foundI = i
+	}
+
+	if foundI >= 0 {
+		return foundI, nil //TODO
```

Comment:
Oops, forgot to remove the TODO. It does work and passes the `builtins` minitest with apropos / partial matching additions.